main.py

```diff
@@ -18,23 +27,51 @@
     help="Take temperature in Celsius and convert to Fahrenheit"
 )
 
+# parser.add_argument(
```

Comment:
@MariaMaximova, it is bad practice to leave commented code.
Please uncomment it and use or remove it.
I suggest uncommenting and trying to get it work as intended to fix the issue with both flags (-c and -f)